From 1d21f15a7b42b02393dfa451297a246807dd55ad Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 16 May 2014 00:13:35 +0300
Subject: [PATCH 4/4] Change vendorarch to include the multiarch triplet

This is currently only a draft. The change will be
implemented in a future perl major version upgrade.
---
 perl-policy.sgml | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/perl-policy.sgml b/perl-policy.sgml
index c23f7c3..63fd730 100644
--- a/perl-policy.sgml
+++ b/perl-policy.sgml
@@ -170,14 +170,18 @@ $Config{sitelib}   (currently /usr/local/share/perl/<var>version</var>)
 	      <p>
 		Packaged modules (see <ref id="module_packages">).
 	        <example>
-$Config{vendorarch} (currently /usr/lib/perl5)
+$Config{vendorarch} (currently /usr/lib/<var>triplet</var>/perl5)
 $Config{vendorlib}  (currently /usr/share/perl5)
 	        </example>
+		Where <var>triplet</var> is the multiarch triplet for
+		the system architecture.
+	      </p>
+	      <p>
 		These locations, particularly
 		<tt>$Config{vendorarch}</tt>, may change if
-		necessary<footnote>For example, to include
-		the multiarch triplet</footnote>.  Packages
-		should use <tt>$Config{vendorlib}</tt> and
+		necessary<footnote>The last change was made
+		to include the multiarch triplet</footnote>.
+		Packages should use <tt>$Config{vendorlib}</tt> and
 		<tt>$Config{vendorarch}</tt>, not hardcode the current
 		locations.<footnote>Build systems based on
 		<tt>ExtUtils::MakeMaker</tt> and <tt>Module::Build</tt>
-- 
2.0.0.rc2

